### PR TITLE
Added Spanish translation

### PR DESCRIPTION
**Pull request by LinkSake**
Hi, I made a version of the Docs in Spanish based on how the other two languages are set up; let me know if it needs some changes.

### CODE REVIEW

**Comment by nelson6e65** [comment]
Hola, @LinkSake . 👋🏻


I was about to start my translation. It looks good to me. Nice work! 😀

I noticed some minor issues only, but I'll check it better this night when I have more time. 🙂

**Comment by LinkSake** [comment]
Awesome! Muchas gracias, @nelson6e65!

File: es-mx/dynamic_analyzer.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+Ejemplo: `ANALYZER_IDENTIFIER = '192.168.56.101:5555'`.
+Se puede encontrar la dirección IP del dispositivo desde Genymotion, y su puerto default es el `5555`.
+
+![IP del Analizador Dinámico](https://user-images.githubusercontent.com/4301109/65379210-0b312300-dce2-11e9-8827-f63d3b95dfd1.png)
+
+## Emulador de Android Studio
+
+?> Tiene soporte para las arquitecturas x86, x86_64 de Android **5.0 - 9.0**, hasta la versión **28 del API**.
+
+La imagen del emulador de Android con Google Play Store está considerada como una imagen de producción, por ello no se puede usar MobSF con ella.
```
Comment (nelson6e65) [inline es-mx/dynamic_analyzer.md:34]:
```suggestion
La imagen del emulador de Android con Google Play Store está considerada como una imagen de producción, por ello no se puede usar MobSF con ella.
Se necesita crear un Dispositivo Android Virtual (AVD) **sin Google Play Store**. No inicies el AVD desde Android Studio, en su lugar inícialo desde la terminal usando la opción `emulador`.
```

File: es-mx/dynamic_analyzer.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+![ADB Connection](https://user-images.githubusercontent.com/4301109/81505878-9b979480-92c0-11ea-9456-32cf5254d381.png)
+
+3. Accede a la máquina virtual de Genymotion Cloud navegando a la IP pública. El usuario por defecto es `genymotion` y la contraseña es el ID de la instancia de EC2. [Más información](https://docs.genymotion.com/paas/8.0/02_Getting_Started/021_AWS.html#create-and-set-up-an-instance).
+
+4. Navega a la configuración y habilita ADB.
+
+![ADB en Genymotion Cloud](https://user-images.githubusercontent.com/4301109/81505975-46a84e00-92c1-11ea-82a5-8912f96849b1.png)
+
+5. Probar la conección a la máquina virtual de Genymotion Cloud de manera local por medio de ADB.
```
Comment (nelson6e65) [inline es-mx/dynamic_analyzer.md:122]:
```suggestion
5. Probar la conexión a la máquina virtual de Genymotion Cloud de manera local por medio de ADB.
```

File: es-mx/extras.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+## API REST
+
+MobSF proporciona una API REST. Se puede acceder a la última versión de la documentación del API desde la aplicación.
+
+***
+## CI/CD
+
+Se puede tomar ventaja del API de MobSF para CI/CD através de diferentes integraciónes.
+
+**Integraciónes para CI/CD de MobSF**
```
Comment (nelson6e65) [inline es-mx/extras.md:13]:
```suggestion
**Integraciones para CI/CD de MobSF**
```

File: es-mx/extras.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+```python
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': DB_DIR,
+    }
+}
+```
+
+Enseguida se deberá de descomentar el siguiente bloque de código:
```
Comment (nelson6e65) [inline es-mx/extras.md:92]:
```suggestion
Seguidamente, deberá de descomentar el siguiente bloque de código:
```

File: es-mx/extras.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+        'PASSWORD': '',
+        'HOST': 'localhost',
+        'PORT': '',
+    }
+}
+```
+
+Después se deberá de crear una base de datos en PosgreSQL llamada `mobsf` y configurarla con las opciónes correctas que aparecen en el bloque de código anterior.
+
+**Aplicar las Migraciónes**
```
Comment (nelson6e65) [inline es-mx/extras.md:109]:
```suggestion
**Aplicar las Migraciones**
```

File: es-mx/extras.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+```bash
+python manage.py makemigrations 
+python manage.py makemigrations StaticAnalyzer
+python manage.py migrate
+```
+
+Finalmente es necesario reiniciar el servidor de MobSF, después de ello se habrá configurado de manera éxitosa la nueva base de datos.
```
Comment (nelson6e65) [inline es-mx/extras.md:117]:
```suggestion
Finalmente, es necesario reiniciar el servidor de MobSF, con lo cual se habrá configurado Postgres como tu nueva base de datos de manera exitosa.
```

File: es-mx/installation.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# Instalación
+
+**Probado en Windows 10, Ubuntu (18.04, 19.04) y macOS Catalina**
+
+!> Se debe de cumplir todos los [requerimientos](/es-mx/requirements.md) previos antes de intentar la instalación de MobSF.
```
Comment (nelson6e65) [inline es-mx/installation.md:5]:
```suggestion
!> Se deben cumplir todos los [requisitos](/es-mx/requirements.md) antes de intentar la instalación de MobSF.
```

File: es-mx/requirements.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
+# Requerimientos Previos
```
Comment (nelson6e65) [inline es-mx/requirements.md:1]:
```suggestion
# Requisitos
```

File: es-mx/requirements.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+* **MacOS**
+  * Instalar [Git](https://www.atlassian.com/git/tutorials/install-git).
+  * Instalar [Python **3.8 o 3.9**](https://www.python.org/).
+  * Después de instalar Python 3.8+: Se debe de ir a `/Applications/Python 3.8/` y ejecutar `Update Shell Profile.command`, después ejecutar `Install Certificates.command`.
+  * Instalar el [JDK 8+](https://www3.ntu.edu.sg/home/ehchua/programming/howto/JDK_Howto.html).
+  * Instalar las command line tools con el comando `xcode-select --install`.
+  * Descargar e instalar [wkhtmltopdf](https://wkhtmltopdf.org/downloads.html) como se indica en las [instrucciónes](https://github.com/JazzCore/python-pdfkit/wiki/Installing-wkhtmltopdf).
+  * Para realizar un análisis estático de una aplicación de Windows se necesita una computadora con Windows o una máquina virtual en Mac o Linux. [Más información](https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/master/mobsf/install/windows/readme.md).
+
+* **Distibuciónes de Linux basadas en Ubuntu/Debian**:
```
Comment (nelson6e65) [inline es-mx/requirements.md:14]:
```suggestion
* **Distribuciones de Linux basadas en Ubuntu/Debian**:
```

File: es-mx/running.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
+# Correr MobSF
```
Comment (nelson6e65) [inline es-mx/running.md:1]:
```suggestion
# Ejecutar MobSF
```

File: es-mx/tests.md [hunk 11]
```diff
@@ -0,0 +1,19 @@
+# Corriendo las Pruebas
```
Comment (nelson6e65) [inline es-mx/tests.md:1]:
```suggestion
# Ejecutando las Pruebas
```

File: es-mx/updating.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
+# Actualizando MobSF
```
Comment (nelson6e65) [inline es-mx/updating.md:1]:
```suggestion
# Actualizar MobSF
```

File: es-mx/updating.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+En la mayor parte de los casos, si se está actualizando MobSF, se deberá de realizar una migración de la base de datos; de lo contrario se podrían presentar errores como el siguiente:
```
Comment (nelson6e65) [inline es-mx/updating.md:3]:
```suggestion
En la mayoría de los casos, si se está actualizando MobSF, se deberá de realizar una migración de la base de datos; de lo contrario se podrían presentar errores como el siguiente:
```

**Review by nelson6e65 — CHANGES_REQUESTED** [review]

**Comment by nelson6e65** [comment]
@LinkSake I added some suggestion and minor fixes. You can use the review “patch” or “batch patches” directly in most of the cases.

**Comment by nelson6e65** [comment]
In the other side, it can be used `es` instead of `es-mx`, due to neutral accent.

> Also, "es-419" represents "Spanish as used in Latin America and the Caribbean".

**Comment by LinkSake** [comment]
> In the other side, it can be used es instead of es-mx, due to neutral accent.
I think you're right! Let me change that real quick.

**Pull request merged**